### ISSUE TITLE
Mob tagging outside of instances gives incorrect (halved) experience after the most recent patch

### ISSUE THREAD

**Issue by Viperlisk** [elysium-project/bug-tracker#361]
Tagging a mob of ~equal level **outside of a party** and having a level 60 player
kill the mob should reward full experience. This was a common leveling/boosting
technique in vanilla and later expansions in WoW before being removed in cataclysm

Result before the latest patch:
Player A (level 20) tags a level 20 mob outside of a party. Player B (level 60) kills the mob.
Player A gets **full** experience (as much experience as if he had killed the mob by himself).

Result after the latest patch:
Player A (level 20) tags a level 20 mob outside of a party. Player B (60) kills the mob.
Player A gets **half** the experience as if he had killed the mob by himself.

The end result of this is that tag leveling guild mates and friends is no longer a viable
method of helping them level up.

Additionally **this potentially allows for griefing** of low level same-faction players by intentionally
halving their experience by killing their mobs outside of their party.

**Comment by Viperlisk** [comment]
After further testing 2 hours after the initial report we were unable to reproduce the bug.

Original data that led to us reporting the issue was the following:

Player A - level 20 druid;
Player B - level 60 warrior;
No party.

Target - level 14 Zhevra Runner in The Barrens

If Player A solo killed the mob it gave 130 XP (including the rested bonus).
If Player A tagged the mob and Player B killed it it gave 64 XP instead (including the rested bonus).

When we tried to get more data later Player A no longer had any rested XP and we were consistently getting 65 XP per mob regardless if done solo or assisted. If done it a party the same mob would award 14 XP.

As such, the issue we reported appears to have been fixed.
